internal/domain/service/file_service_test.go

```
@@ -1300,7 +1307,10 @@ func (m *dedupMockStorage) Save(content []byte) (model.StoredFile, error) {
 	return model.StoredFile{ExternalID: hash, Size: len(content), Created: false}, nil
 }
 func (m *dedupMockStorage) Read(id string) ([]byte, error) { return m.inner.Read(id) }
-func (m *dedupMockStorage) Delete(_ string) error          { m.deleted = true; return nil }
+func (m *dedupMockStorage) ReadStream(id string) (io.ReadCloser, int64, error) {
+	return m.inner.ReadStream(id)
+}
+func (m *dedupMockStorage) Delete(_ string) error { m.deleted = true; return nil }
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)